pkg/cli/user.go

```diff
@@ -106,3 +116,67 @@ type Role struct {
 	RoleName   string
 	Privileges []string
 }
+
+// createCert creates a key pair for login purpose
+func (u *User) createCert() error {
+	id := strings.TrimeSuffix(u.cert, ".crt")
+	var encodedCert bytes.Buffer
+	certFile, err := os.Create(id + ".crt")
+	if err != nil {
+		return err
+	}
+	defer certFile.Close()
+
+	keyFile, err := os.Create(id + ".key")
+	if err != nil {
+		return err
+	}
+	defer keyFile.Close()
+
+	priv, err := rsa.GenerateKey(rand.Reader, 2048)
+	if err != nil {
+		return err
+	}
+
+	notBefore := time.Now()
+	notAfter := notBefore.Add(5 * 365 * 24 * time.Hour) // 5 years
+
+	serialNumberLimit := new(big.Int).Lsh(big.NewInt(1), 128)
+	serialNumber, err := rand.Int(rand.Reader, serialNumberLimit)
+	if err != nil {
+		return err
+	}
+
+	template := x509.Certificate{
+		SerialNumber: serialNumber,
+		Subject: pkix.Name{
+			Organization: []string{"com.vmware.vsphere-cloud-provider"},
```

Comment:
What should the org be, I just make an sample here..

Reply:
How about just `"sig-vmware"`?

Reply:
I would suggest a more generic 'kubernetes'